neuralforecast/common/_base_recurrent.py

```diff
@@ -96,7 +123,15 @@ def on_fit_start(self):
         random.seed(self.random_seed)
 
     def configure_optimizers(self):
-        return torch.optim.Adam(self.parameters(), lr=self.learning_rate)
+        optimizer = torch.optim.Adam(self.parameters(), lr=self.learning_rate)
+        scheduler = {
+            "scheduler": torch.optim.lr_scheduler.StepLR(
+                optimizer, self.lr_decay_steps, 0.5
```

Comment:
to have better code readability and protect us agains inputs ordering changes
can we add the name of the arguments?

https://pytorch.org/docs/stable/generated/torch.optim.lr_scheduler.StepLR.html

Reply:
Done